### DISCUSSION TITLE
In-place upgrade from Windows 11 to 11 with skipping checks

### DISCUSSION THREAD

**Discussion by Nicos18** [AveYo/MediaCreationTool.bat#137] [Q&A] [answered]
Hi,

I need to do an in-place upgrade keeping all my files and apps with just skipping the TPM check: can I use the auto.cmd script as it is, or do I need to change something?

Thanks

**Comment by AveYo** [comment, marked as answer]
yes, you can, it's why I've shared it as stand-alone
